From e634ad5bc6022e3f63bb4d48b796e78e11053c48 Mon Sep 17 00:00:00 2001
From: David Steele <dsteele@gmail.com>
Date: Mon, 7 Oct 2013 18:25:36 -0400
Subject: [PATCH 13/20] Variable spelling errors in xmpp.py exception handling.

Missing 'n' in XmppConnection self._conected.
(cherry picked from commit 7fa4647dfc38e4272e72fcb4d46c8fb2bec66c6c)
---
 cloudprint/xmpp.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cloudprint/xmpp.py b/cloudprint/xmpp.py
index 41f4728..a1b83cf 100644
--- a/cloudprint/xmpp.py
+++ b/cloudprint/xmpp.py
@@ -57,7 +57,7 @@ class XmppConnection(object):
                 # socket closed
                 raise Exception("xmpp socket closed")
         except Exception:
-            self._conected = False
+            self._connected = False
             raise
 
         LOGGER.debug('<<< %s' % data)
@@ -71,7 +71,7 @@ class XmppConnection(object):
             self._wrappedsock.sendall(msg)
             self._nextkeepalive = time.time() + self._keepalive_period
         except Exception:
-            self._conected = False
+            self._connected = False
             raise
 
     def _msg(self, msg=None):
-- 
1.8.4.rc3

